apps/storefront/src/app/[locale]/(checkout)/checkout/(checkout-details)/payment/address-tab.tsx

```diff
@@ -53,6 +53,10 @@ export function AddressTab({
   const form = useFormContext();
   const region = useCurrentRegion();
 
+  const hasDefaultBillingAddressSet = addresses?.some(
+    (item) => item.address?.isDefaultBillingAddress === true,
+  );
```

Comment:
Just
```suggestion
  const hasDefaultBillingAddressSet = addresses?.some(
    (item) => item.address?.isDefaultBillingAddress,
  );
```
since it's a bool prop.